RELEASE.md

```diff
@@ -17,6 +17,7 @@
 ## Breaking changes to the API
 
 ## Thanks for supporting contributions
+[Lovkush Agarwal](https://github.com/Lovkush-A)
```

Comment:
Could you move this to the section under `0.16.4` please?

Reply:
I have moved it to section 0.16.4.

Reply:
0.16.4 was already released, and your contribution is going to be included in the upcoming 0.16.5 release, so please add your name under 0.16.5 instead of 0.16.4. Let me know if you need help with this :)

Reply:
Alright, done!